text/0000-decorators.md

```diff
@@ -0,0 +1,1230 @@
+- Start Date: 2018-10-20
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Decorators
+
+## Summary
+
+Native classes are now officially supported in Ember, but currently their usage
+is very limited. Core Ember features such as computed properties, actions, and
+service injections have no publicly supported APIs to be used with native class
+syntax.
+
+Decorators provide a way to abstract functionality and improve the developer
+experience of working with native classes.  This RFC outlines the implementation
+and rollout plan for bringing decorators to Ember's computed properties (and
+other behavior) for use in native classes.
+
+### A Note on Decorator Stability
+
+Decorators are important to adopting native class syntax. They are a
```

Comment:
you may want to mention something about "stuff that belongs on prototypes" as part of the motivation for decorators